packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js

```diff
@@ -79,12 +90,14 @@ export default class ScrubbingCenterController extends AbstractCursorDatagridCon
   }
 
   selectService() {
+    this.selectedIp = null;
+    this.isEmpty = false;
+    this.model = null;
     if (this.service) {
       this.pageSize = 10;
       this.page = 1;
       this.autocomplete = [];
       this.ipsList = [];
-      this.selectedIp = '';
       this.ipsList = null;
```

Comment:
this.ipsList is assigned twice as null and then [], since it should be an array later, maybe just keep this.ipsList = []; 
```suggestion
```